patches/net/minecraft/client/gui/Gui.java.patch

```diff
@@ -20,8 +20,19 @@
 +    private final net.neoforged.neoforge.client.gui.GuiLayerManager layerManager = new net.neoforged.neoforge.client.gui.GuiLayerManager();
      protected float scopeScale;
  
++    /**
++     * Neo: This variable controls the height of overlays on the left of the hotbar (e.g. health, armor).
++     */
++    public int leftHeight = 39;
```

Comment:
IMO we should only initialize these to 39 in one spot. Is there a reason why we need to do it both here and in `render`?

Reply:
Also, is there any constant in vanilla we can use to retrieve 39 instead of hardcoding it?

Reply:
There is no such constant sadly. The field initialization is quite useless, I'll remove it. I copied it over from `ExtendedGui`.